dev-token.py

```diff
@@ -0,0 +1,30 @@
+import argparse
+import getpass
+import sys
+from datetime import datetime, timedelta, timezone
+from urllib.parse import urljoin
+
+from hatch import config, signing
+
+
+def token(argv=sys.argv):
+    parser = argparse.ArgumentParser()
+    parser.add_argument("workspace", help="workspace for token url")
```

Comment:
The docs mention this isn't required but, and my argparse-memories are very rusty, isn't this a required argument?

Reply:
huh, yeah, I messed this up, will fix.

Not sure I want to allow an unrestricted token, so will required it

Reply:
I removed the incorrect doc, workspace is required